rasa/telemetry.py

```diff
@@ -370,40 +395,29 @@ async def _send_event(
 
     headers = segment_request_header(write_key)
 
-    async with aiohttp.ClientSession() as session:
-        async with session.post(
-            SEGMENT_ENDPOINT, headers=headers, json=payload,
-        ) as resp:
-            # handle different failure cases
-            if resp.status != 200:
-                response_text = await resp.text()
-                logger.debug(
-                    f"Segment telemetry request returned a {resp.status} response. "
-                    f"Body: {response_text}"
-                )
-            else:
-                data = await resp.json()
-                if not data.get("success"):
-                    logger.debug(
-                        f"Segment telemetry request returned a failure. "
-                        f"Response: {data}"
-                    )
-
-
-def _project_hash() -> Text:
-    """Create a hash for the project in the current working directory.
+    resp = requests.post(SEGMENT_ENDPOINT, headers=headers, json=payload)
+    # handle different failure cases
+    if resp.status_code != 200:
+        logger.debug(
+            f"Segment telemetry request returned a {resp.status_code} response. "
+            f"Body: {resp.text}"
+        )
+    else:
+        data = resp.json()
+        if not data.get("success"):
+            logger.debug(
+                f"Segment telemetry request returned a failure. " f"Response: {data}"
+            )
+
+
+def _hash_directory_path(path: Text) -> Optional[Text]:
+    """Create a hash for the directory.
 
     Returns:
-        project hash
+        hash of the directories path
     """
-    try:
-        remote = check_output(  # skipcq:BAN-B607,BAN-B603
-            ["git", "remote", "get-url", "origin"], stderr=DEVNULL
-        )
-        return hashlib.sha256(remote).hexdigest()
-    except (CalledProcessError, OSError):
-        working_dir = Path(os.getcwd()).absolute()
-        return hashlib.sha256(str(working_dir).encode("utf-8")).hexdigest()
+    working_dir = Path(path).absolute()
```

Comment:
in practice it's the working directory, but just looking at the function it could be any dir